Makefile

```diff
@@ -26,7 +25,7 @@ $(BIN)/license-headers: Makefile
 
 .PHONY: build
 build: generate ## Build the entire project.
-	./gradlew $(GRADLE_ARGS) build -x test
```

Comment:
@smallsamantha - I noticed there were real compile errors in the tests which weren't getting reported in any of the CI checks as part of #79. This should resolve that.